clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -1020,7 +1020,7 @@ export const CustomSwitch = ({
       <FormControl isInvalid={isInvalid} width="full">
         <Box display="flex" alignItems="center" justifyContent="space-between">
           <HStack spacing={1}>
-            <Label htmlFor={props.id || props.name} fontSize="sm" my={0} mr={0}>
+            <Label htmlFor={props.id || props.name} fontSize="xs" my={0} mr={0}>
```

Comment:
I checked and the only fields that use this stacked switch variant are the ones in the privacy declaration form, so this seemed safe to change (the rest of the forms on that page also use `xs`)

Reply:
This change feels to broad. It looks ok on the privacy notice form. But it also affects the system info form and I'm not sure it makes sense there. 
![Screenshot 2023-10-26 at 11 15 02 AM](https://github.com/ethyca/fides/assets/101993653/40535f34-2ff8-4d27-ae51-06388e7869f5)
![Screenshot 2023-10-26 at 11 15 52 AM](https://github.com/ethyca/fides/assets/101993653/a05dc101-c270-448d-ae4f-bbab57a35cec)

Reply:
might I suggest adding fontSize as an optional prop and have the default be small but then pass in xs for this one instance?

Reply:
via Slack, @Kelsey-Ethyca and I decided we need to standardize all the form inputs to 12px (`xs`). here's what that looks like for the system form:

![image](https://github.com/ethyca/fides/assets/24641006/80f05257-b4b9-488d-bee0-06fb04f0c4dc)

And for privacy declaration form:
<img width="739" alt="image" src="https://github.com/ethyca/fides/assets/24641006/53159170-3ae1-4f4e-8363-b6f9ec85e98b">

Reply:
Thank you Allison!